test/e2e_node/README.md

```diff
@@ -0,0 +1,109 @@
+  
+2. If running against non-gce host
+  * *Requires password-less ssh and sudo access*
+  * Make sure you can ssh into the host and run sudo `ssh <hostname> -- sudo ls`
+    * If ssh flags are required (e.g. `-i`), they can be passed to the tests using `--ssh-options`
+  * `godep go run test/e2e_node/runner/run_e2e.go --logtostderr --hosts <comma separated hosts>`
+  
+3. Manually copying the test tar.gz and running the tests
+  * Build the tar.gz `godep go run test/e2e_node/runner/run_e2e.go --logtostderr --build-only`
+  * Copy `e2e_node_test.tar.gz` to the remote host
+  * Extract the archive on the remote host `tar -xzvf e2e_node_test.tar.gz`
+  * Run the tests `./e2e_node.test --logtostderr --v 2 --build-services=false --node-name=<hostname>`
+    * Note: This must be run from the directory containing the kubelet and kube-apiserver binaries.
+
+## Running tests against a gce image
+
+* Build a gce image from a prepared gce host
+  * Create the host from a base image and configure it (see above)
+    * Run tests against this remote host to ensure that it is setup correctly before creating the image
+  * Create a gce *snapshot* of the instance
+  * Create a gce *disk* from the snapshot
+  * Create a gce *image* from the disk
+* Test that your gcloud credentials are setup for the project
+  * `gcloud compute --project <project> --zone <zone> images list`
+  * Verify that your image appears in the list
+* Copy test/e2e_node/jenkins/template.properties
+  * Fill in GCE_PROJECT, GCE_ZONE, GCE_IMAGES
+* Run `test/e2e_node/jenkins/e2e-node-jenkins.sh <path to properties file>`
+
+## Kubernetes Jenkins CI and PR builder
+
+Node e2e tests are run against a static list of host environments continuously and on every github.com pull request.
+
+### CI Host environments
+
+### PR builder host environments
+
+TBD
+
+### PR builder host environments
+
+| linux distro    | distro version | docker version | etcd version | cloud provider | status       |
+|-----------------|----------------|----------------|--------------|----------------|--------------|
+| containervm     | 7              | 1.8            |              | gce            | disabled     |
+| rhel            | 7              | 1.10           |              | gce            | disabled     |
+| centos          | 7              | 1.10           |              | gce            | disabled     |
+| coreos          | stable         | 1.8            |              | gce            | disabled     |
+| debian          | jessie         | 1.10           |              | gce            | disabled     |
+| ubuntu          | trusty         | 1.8            |              | gce            | passing      |
+| ubuntu          | trusty         | 1.9            |              | gce            | passing      |
+| ubuntu          | trusty         | 1.10           |              | gce            | passing      |
+| ubuntu          | wily           | 1.10           |              | gce            | *failing*    |
```

Comment:
Yeah maybe it isn't worth having.  My intent was to document if there are distros that we know don't work and need someone to look at.  May be this would be better as a list of issues numbers?  I'll delete for now, and we can add if needed at some point.